src/js/common/schemaform/SaveFormLink.jsx

```diff
@@ -63,9 +72,18 @@ class SaveFormLink extends React.Component {
             user={this.props.user}
             onUpdateLoginUrl={this.props.onUpdateLoginUrl}
             onLogin={this.saveFormAfterLogin}/>
-        {savedStatus === SAVE_STATUSES.failure &&
-          <div className="usa-alert usa-alert-error no-background-image schemaform-save-error">We’re sorry, but something went wrong. Please try saving your application again.</div>}
-        {content}
+        {saveErrors.has(savedStatus) &&
+          <div role="alert" className="usa-alert usa-alert-error no-background-image schemaform-save-error">
```

Comment:
It seems like we're inconsistent about that. Something worth bringing up to the designers, since the icon is the default styling from uswds, but in certain spots where the text in the alert doesn't have paragraph or line breaks this more compact format seems to fit better.